src/aiperf/common/config/video_config.py

```
@@ -29,11 +28,28 @@ def validate_width_and_height(self) -> Self:
             raise ValueError("Height is specified but width is not")
         return self
 
+    @model_validator(mode="after")
+    def _validate_video_options(self) -> Self:
+        """Validate the video options."""
+        video_options_set = self.model_fields_set
+        if not self.videos_enabled() and video_options_set:
+            raise ValueError(
+                "Video generation is disabled but video options were provided. Please set `--video-width` and `--video-height` to enable video generation."
+            )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)